examples/sbol2/CreLoxRecombination.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Representing Cre-Lox Recombination with SBOL2\n",
+    "\n",
+    "Cre-Lox recombination is a site-specific recombination event, often used to modify sequences in synthetic biology by flipping or excising segments of DNA. This can be used to activate or deactivate genes or other elements.\n",
+    "\n",
+    "In this example, we will represent a situation where an unknown coding sequence (CDS) is initially in the wrong orientation and flanked by two loxP recombination sites. Upon recombination, the CDS will be flipped into the correct orientation. We'll use SBOL2 concepts like `Component`, `SequenceAnnotation`, `Range`, `GenericLocation`, and `SequenceConstraint` to model this event."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sbol2\n",
+    "\n",
+    "# Create an SBOL document\n",
+    "doc = sbol2.Document()\n",
+    "\n",
+    "# Set a namespace for the document\n",
+    "sbol2.setHomespace('https://github.com/SynBioDex/SBOL-Notebooks')\n",
+    "\n",
+    "# Create a ComponentDefinition for the entire Cre-Lox recombination system\n",
+    "recombination_system = sbol2.ComponentDefinition('CreLox_Recombination_System', sbol2.BIOPAX_DNA)\n",
+    "doc.addComponentDefinition(recombination_system)\n",
+    "\n",
+    "# Create the Components for the Cre-Lox recombination sites\n",
+    "# Component for the first loxP site\n",
+    "loxP1 = sbol2.ComponentDefinition('loxP_site_1', sbol2.BIOPAX_DNA)\n",
+    "loxP1_seq = sbol2.Sequence('loxP_seq_1', 'ATAACTTCGTATAATGTATGCTATACGAAGTTAT', sbol2.SBOL_ENCODING_IUPAC)\n",
+    "loxP1.sequences = [loxP1_seq.identity]\n",
+    "doc.addComponentDefinition(loxP1)\n",
+    "\n",
+    "# Component for the second loxP site\n",
+    "loxP2 = sbol2.ComponentDefinition('loxP_site_2', sbol2.BIOPAX_DNA)\n",
+    "loxP2_seq = sbol2.Sequence('loxP_seq_2', 'ATAACTTCGTATAATGTATGCTATACGAAGTTAT', sbol2.SBOL_ENCODING_IUPAC)\n",
+    "loxP2.sequences = [loxP2_seq.identity]\n",
+    "doc.addComponentDefinition(loxP2)\n",
+    "\n",
+    "# Create a Component for the CDS (Coding Sequence)\n",
```

Comment:
```suggestion
    "# Create a ComponentDefinition for the CDS (Coding Sequence)\n",
```

A side note, which you might or might not want to say anything about: in SBOL3 this isn't required, because we can use a constraint on a SequenceFeature. It's so much nicer that way, because we don't have to have dummy TopLevel objects floating around.